lib/engines/content_block_manager/app/assets/javascripts/content_block_manager/modules/copy-embed-code.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+;(function (Modules) {
+  function CopyEmbedCode(module) {
+    this.module = module
+    this.copyLink = document.createElement('a')
```

Comment:
nitpick/accessibility: should this be a `button` element rather than an anchor, as it's doing something rather than taking us somewhere? or have the `aria-role` of button? not sure if it makes much of a difference for screen readers though tbh.

Reply:
Yeah, I've added `role=button` to this now - ta!